pkg/framework/guest.go

```diff
@@ -105,32 +106,63 @@ func (g *Guest) Setup() error {
 }
 
 func (g *Guest) WaitForAPIDown() error {
-	apiDown := func() error {
-		_, err := g.k8sClient.
-			CoreV1().
-			Services("default").
-			Get("kubernetes", metav1.GetOptions{})
-
-		if err == nil {
-			return microerror.Mask(fmt.Errorf("API up"))
+	time.Sleep(1 * time.Second)
```

Comment:
I just added this arbitrary wait second to prevent cases in which the API is still available and this the check below succeeds unexpectedly.